Tasks/PublishSymbolsV2/task.json

```diff
@@ -160,18 +169,35 @@
         }
     ],
     "instanceNameFormat": "Publish symbols path",
+    "prejobexecution": {
+        "Node10": {
+            "target": "PreJobExecutionPublishSymbols.js"
+        }
+    },    
     "execution": {
         "PowerShell3": {
-            "target": "PublishSymbols.ps1"
+            "target": "PublishSymbols.ps1",
+            "platforms": [
+                "windows"
+            ]
+        },
+        "Node10": {
+            "target": "PublishSymbols.js"
```

Comment:
I believe this should be clienttoolmain.js since clienttoolmain.ts is the file that actually seems to run something rather than just exporting functions